packages/api/src/command/medical/patient/reset-external-data.ts

```diff
@@ -0,0 +1,42 @@
+import { MedicalDataSource, isMedicalDataSource } from "@metriport/core/external/index";
+import { BadRequestError } from "@metriport/shared";
+import { PatientModel } from "../../../models/medical/patient";
+import { getPatientModelOrFail } from "./get-patient";
+import { executeOnDBTx } from "../../../models/transaction-wrapper";
+
+export type ResetExternalData = {
+  cxId: string;
+  patientId: string;
+  source: string;
+};
+
+export const resetExternalData = async ({
```

Comment:
no const functions please

also, super important function!
- let's add the "ADMIN ONLY" TSDoc we have for functions like this.
- and move it to the admin folder (`packages/api/src/command/medical/admin`)?